docs/tutorials/setting-up.md

```diff
@@ -110,8 +127,8 @@ own). Here is how your DNS zone should look like:
 example.org.   A     10.2.3.4
```

Comment:
Probably these need to be updated as well to mention mx1

Reply:
Done